types/leaflet-responsive-popup/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
strictness cannot be turned off, please turn on both flags and fix errors if any

Reply:
Thank you for your review. I changed the file.